modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -231,6 +231,8 @@ A periodic housekeeping task in Redpanda performs compaction and removes data th
 
 * When `retention_local_strict` is false (default), the housekeeping process removes data above the configured consumable retention. This means that data usage is allowed to expand to occupy more of the log data reservation.
 * When `retention_local_strict` is true, the housekeeping process uses local retention settings to select what data to remove.
++
+NOTE: By default, `retention_local_strict` is set to false in new clusters and set to true in upgraded clusters.
```

Comment:
> upgraded clusters

Does this only apply to the 23.2 doc? Once we release 23.3, this doesn't apply to the upgraded cluster from 23.2 to 23.3 where 23.2 was a new cluster.

If it's ok to mention about specific version in the doc, can we simply say like below?

> and set to true in upgraded clusters from 23.1 and earlier.

Reply:
Thanks, Daisuke! @dotnwat can you please confirm if this is correct to add to doc: 
The `retention_local_strict` property is set to true in clusters upgraded from release 23.1 and earlier.

Reply:
lgtm